client/components/TestRenderer/index.jsx

```diff
@@ -0,0 +1,875 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import nextId from 'react-id-generator';
+import {
+    userCloseWindow,
+    userOpenWindow
+} from '../../resources/aria-at-test-run.mjs';
+import {
+    TestRunExport,
+    TestRunInputOutput
+} from '../../resources/aria-at-test-io-format.mjs';
+import { TestWindow } from '../../resources/aria-at-test-window.mjs';
+
+const Container = styled.div`
```

Comment:
The benefit of using styled components may not come in the short term. Would rather to revert to using a custom stylesheet for this component

Reply:
I like StyledComponents 🤓 I'd be down to gradually move over to it if you like.

Reply:
Following an internal discussion, we'll be moving forward with styled components as it may promote ease of maintainability